doc/Development_Documentation/05_Objects/01_Object_Classes/05_Class_Settings/26_Interfaces_and_traits.md

```diff
@@ -0,0 +1,74 @@
+# Using interfaces and traits
+In some cases you need to implement interfaces and overwrite some provided functions.
```

Comment:
```suggestion
In some cases it could be helpful to let the generated PHP class for data objects implement interfaces or add some additional functions using traits.
```